### ISSUE TITLE
Sort and Search & Export

### ISSUE THREAD

**Issue by 00gnarlylion** [Rplus/Pokemon-shiny#137]
Having the ability to sort the list, Search the list, or export to a CSV format would be nice too.

**Comment by hamsicik** [comment]
I think so... We need filters and the names of Pokémons.

**Comment by olehsamoilenko** [comment]
True, export feature is needed (in CSV or string) to share the list of needed Pokemon to a friend.

**Comment by Rplus** [comment]
@olehsamoilenko 
Maybe you could try to share the list with friends by this link?
![image](https://user-images.githubusercontent.com/1808835/233117140-b3c4cd9b-84cb-4c8b-b684-f21070cebaca.png)
